tier4_planning_msgs/msg/PlanningFactor.msg

```diff
@@ -0,0 +1,32 @@
+# msg format version
+string version_msg_format
+
+# constants for common use
+uint16 UNKNOWN = 0
+
+# constants for longitudinal behavior
+uint16 SLOW_DOWN = 1
+uint16 STOP = 2
+uint16 GO = 3
+
+# constants for lateral behavior
+uint16 LEFT_SHIFT = 1
+uint16 RIGHT_SHIFT = 2
+uint16 STRAIGHT = 3
+
+# variables
+string module
+
+bool is_driving_forward
+
+tier4_planning_msgs/ControlPoint[] velocity_control_points
+
+tier4_planning_msgs/ControlPoint[] steering_control_point
+
+uint16 velocity_behavior
+
+uint16 steering_behavior
```

Comment:
Isn't a behavior required for each ControlPoint? For example, when avoiding a street parking situation, there are two Control Points, the first for right shift and the second for left shift.

Reply:
I assume that we create PlanningFactor per each shifts, and push the two PlanningFactors into PlanningFactorArray. So, I don't think each ControlPoints needs *_behavior.